src/tools/compiletest/src/runtest.rs

```diff
@@ -1816,12 +1826,22 @@ impl<'test> TestCx<'test> {
         }
     }
 
-    fn dump_output(&self, out: &str, err: &str) {
+    fn dump_output(&self, print_output: bool, proc_name: &str, out: &str, err: &str) {
         let revision = if let Some(r) = self.revision { format!("{}.", r) } else { String::new() };
 
         self.dump_output_file(out, &format!("{}out", revision));
         self.dump_output_file(err, &format!("{}err", revision));
-        self.maybe_dump_to_stdout(out, err);
+
+        if !print_output {
+            return;
+        }
+
+        let proc_name = Path::new(proc_name).file_name().unwrap().to_string_lossy();
+        eprintln!("------{proc_name} stdout------------------------------");
+        eprintln!("{}", out);
+        eprintln!("------{proc_name} stderr------------------------------");
+        eprintln!("{}", err);
+        eprintln!("------------------------------------------");
```

Comment:
Suggestion: can you revert all of these to `println` as we dump libtest output to stderr? This will probably conflict with #134207 but yeah.

Reply:
i'm going to wait to make changes here until #134207 lands. but yes, i'll make sure it's just a code move and not changing the output stream.